test/lib/dependency-versions-test.ts

```diff
@@ -51,6 +51,13 @@ describe('Utils | dependency-versions', function () {
                     'package1'
                   ),
                 }),
+                expect.objectContaining({
```

Comment:
I'd like to request some changes to the tests.

1. Since the new logic is contained to `getPackages` in `workspace.ts`, we can actually add all the new tests in `workspace-test.ts` instead of modifying `dependency-version-test.ts`:

```
describe('#getPackages', function () {
  it('behaves correctly with nested workspaces', function () {
    // new tests go here
  });
});
```

2. Let's avoid touching any existing test fixtures. I think it's becoming too complicated when the existing test fixtures try to showcase too many different scenarios. Instead, I now prefer to have a new test fixture for each new scenario. So I would recommend just creating a minimal nested workspaces test fixture to use in the new test.

Reply:
Fixed.